google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BigtableTracer.java

```diff
@@ -93,4 +93,12 @@ public void grpcChannelQueuedLatencies(long queuedTimeMs) {
   public void grpcMessageSent() {
     // noop
   }
+
+  /**
+   * Set the customer specified deadline so that it can be exported in a metric. This will be
+   * called in BuiltinMetricsTracer.
+   */
+  public void setDeadline(long deadline) {
```

Comment:
maybe rename it to something like "recordRemainingDeadline"? "setDeadline" sounds like it's setting deadine on the export or something :)